### PR TITLE
Python plugins

### PR DESCRIPTION
**Pull request by pelijah**

### CODE REVIEW

File: src/MainWindow.cpp [hunk 1]
```diff
@@ -191,46 +192,49 @@ void MainWindow::initUI()
      */
     dockWidgets.reserve(20);
 
-    disassemblyDock = new DisassemblyWidget(this, ui->actionDisassembly);
-    sidebarDock = new SidebarWidget(this, ui->actionSidebar);
-    hexdumpDock = new HexdumpWidget(this, ui->actionHexdump);
-    pseudocodeDock = new PseudocodeWidget(this, ui->actionPseudocode);
-    consoleDock = new ConsoleWidget(this, ui->actionConsole);
+#define DOCK(w, a) (new CutterDockWidget(this, w, a))
```
Comment (xarkes) [inline src/MainWindow.cpp]: I think I did that, I don't remember, but I am not very satisfied with it.

Comment (pelijah) [reply src/MainWindow.cpp]: It was @thestr4ng3r

File: src/MainWindow.cpp [hunk 2]
```diff
@@ -963,7 +958,7 @@ void MainWindow::on_actionExport_as_code_triggered()
     cmdMap[filters.last()] = "pcp";
     filters << tr("Print 'wx' r2 commands (*.r2)");
     cmdMap[filters.last()] = "pc*";
-    filters << tr("GAS .byte blob (*.asm, *.s)");
+    filters << tr("GAS .byte blob (*.txt)");
```
Comment (xarkes) [inline src/MainWindow.cpp]: Actually the ".asm, .s" string is better (it's the one on master at least)

Comment (Maijin) [reply src/MainWindow.cpp]: How's this related to python plugin, probably better in a separated commit

Comment (xarkes) [reply src/MainWindow.cpp]: It's the rebase, he just took the wrong part, that happens.

Comment (Maijin) [reply src/MainWindow.cpp]: kk

**Comment by XVilka** [comment]
Seems like Travis stuck on this one.

**Comment by XVilka** [comment]
What was desided with this one? Note that since Qt 5.12 there is an official python support, no need for PySide2: 
- http://blog.qt.io/blog/2018/11/15/python-qt-3000-hours-developer-insight/
- https://www.qt.io/qt-for-python

**Comment by thestr4ng3r** [comment]
PySide2 IS the official python support from Qt.

**Comment by XVilka** [comment]
OK, sorry, misunderstood then their release announce.

**Comment by rdlohith** [comment]
src/MainWindow.cpp

**Pull request closed without merging**